README.md

```diff
@@ -1,70 +1,52 @@
-# Getting Started with Create React App
+# Metrics-webapp
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A web application that checks a list of metrics created the data fetched from a covid 19 API for countries in North America.This project applies HTML, CSS, and JavaScript best practices using React/Redux library and is fully responsive.
 
-## Available Scripts
+## live demo
+[loom video](https://www.loom.com/share/2ffaa5fca0fe46a0b029ea1a7623d5fd)
```

Comment:
- [ ] Please, consider adding some words to highlight a piece of the code that you find interesting or are proud of making, as it was requested by previous reviewer 👍

Reply:
thanks for your feedback,I just  applied a new video